### PR TITLE
Add a pause-play test page to protect against http://crbug.com/718369.

### PR DESCRIPTION
**Pull request by ehlemur-zz**
Adds a test page that creates `numPeerConnections` and randomly plays/pauses an stream every `pausePlayIterationDelayMillis` milliseconds.
This in order to prevent regressions like the [freeze bug](http://crbug.com/718369).

### CODE REVIEW

File: src/pause-play/js/main.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+class FeedTable {
+  constructor() {
+    this.numCols = 5;
+    this.col = 0;
+    this.testTable = document.getElementById('test-table');
+    this.row = this.testTable.insertRow(-1);
+  }
+
+  addNewCell() {
+    if (this.col == this.numCols) {
```
Comment (KaptenJansson) [inline src/pause-play/js/main.js]: Use strict type equal operator `===`.

File: src/pause-play/js/main.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    this.remoteConnection = null;
+    this.remoteView = videoElement;
+  }
+
+  start() {
+    const onGetUserMediaSuccess = this.onGetUserMediaSuccess.bind(this);
+    return navigator.mediaDevices
+        .getUserMedia({
+          audio: true,
+          video: {
```
Comment (KaptenJansson) [inline src/pause-play/js/main.js:66]: Use modern constraints patterns, check example 9 in the [spec](https://w3c.github.io/mediacapture-main/getusermedia.html#interface-definition).

File: src/pause-play/js/main.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    this.localConnection.setRemoteDescription(desc);
+  };
+
+  onIceCandidate(connection, event) {
+    if (event.candidate) {
+      connection.addIceCandidate(new RTCIceCandidate(event.candidate));
+    }
+  };
+}
+
```
Comment (KaptenJansson) [inline src/pause-play/js/main.js:113]: Remove two lines.

File: src/pause-play/js/main.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    this.lastIterationTime = Date.now();
+    $('status').textContent = status
+    if (status != 'ok-done') {
+      setTimeout(
+          () => this.pauseAndPlayLoop(), this.pausePlayIterationDelayMillis);
+    }
+  }
+
+  getStatus() {
+    if (this.iteration == 0) {
```
Comment (KaptenJansson) [inline src/pause-play/js/main.js]: Use strict type equal operator `===`.

File: src/pause-play/js/main.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+      if (Math.random() >= 0.5) {
+        feed.play();
+      } else {
+        feed.pause();
+      }
+    });
+    const status = this.getStatus();
+    this.lastIterationTime = Date.now();
+    $('status').textContent = status
+    if (status != 'ok-done') {
```
Comment (KaptenJansson) [inline src/pause-play/js/main.js]: Use strict type not equal operator `!==`.

**Review by KaptenJansson — CHANGES_REQUESTED** [review]

**Comment by ehlemur-zz** [comment]
PTAL

**Review by KaptenJansson — APPROVED** [review]

**Pull request merged**